proof/crefine/autocorres-test/AutoCorresTest.thy

```diff
@@ -132,11 +132,6 @@ local_setup \<open>AutoCorresModifiesProofs.new_modifies_rules "../c/build/$L4V_
 
 text \<open>Extra corres_underlying rules.\<close>
 
-lemma corres_add_noop_rhs2:
```

Comment:
you're modifying crefine but I didn't see any `crefine` tag for any of the commits

Reply:
I can make the tag for this commit `lib+drefine+crefine:`. I did have a look through the commit message guide again, and these points weren't explicitly mentioned, but my thinking here was that it's just moved up to `lib`, and to me, the tag indicates where significant changes are made, and which is the highest session that has been changed, so that we know what would need to be rebuilt.

Reply:
See my comment on the other one. it shouldn't be "add" when you're moving. If it's not significant I'd punt it and say `lib+proof: move ... to ...`
@lsf37 can also weigh in, the moving to lib case is never quite clear.

Reply:
I don't have a strong preference either way